rewrite-java-tck/src/main/java/org/openrewrite/java/tree/VariableDeclarationsTest.java

```diff
@@ -296,27 +295,52 @@ void test(Unknown b) {
       "j",
       "(j)",
       "vari",
-      "(vari)"
+      "(vari)",
+      "var",
+      "(var)",
+      "(var var)",
+      "(Object var)"
     })
-    void lambdaParameterVariableDeclarations(@Language("java") String variableDeclaration) {
-        @Language("java")
-        String java = String.format(
-          """
-            import java.util.function.Consumer;
-            
-            class Foo {
-                void main() {
-                    Consumer<Object> i = %s -> {};
-                }
-            }
-            """,
-          variableDeclaration
+    void lambdaParameterVariableDeclarations(String variableDeclaration) {
+        rewriteRun(
+          java(
+            """
+              import java.util.function.Consumer;
+              
+              class Foo {
+                  void main() {
+                      Consumer<Object> i = %s -> {};
+                  }
+              }
+              """.formatted(variableDeclaration)
+          )
         );
+    }
+
+    @MinimumJava11
+    @ParameterizedTest
+    @ValueSource(strings = {
+      "i, j",
+      "i, var",
+      "var, i",
+      "var i, var j",
+      "var i, var var",
+      "var var, var i"
```

Comment:
@sambsnyd wouldn't the case you highlighted be handled by these tests?

Reply:
Like you I want to make sure this works as well, even if it's a fairly niche cache for Java 11 variables named `var`. Just to double check I looked whether the tests ran for Java 11, and that looks to be the case:
https://ge.openrewrite.org/s/lwie2wx2okql4/tests/task/:rewrite-java-11:compatibilityTest/details/org.openrewrite.java.tree.VariableDeclarationsTest?top-execution=1